Makefile

```diff
@@ -89,7 +76,7 @@ test-spec:
 	tools/update_spec --check
 
 .PHONY: test-static
-test-static: tidy-check test-yaml-valid test-merge test-dry test-no-wait_idle test-unused-modules test-soft_failure-no-reference test-spec test-invalid-syntax
+test-static: tidy-check test-yaml-valid test-metadata-changed test-dry test-no-wait_idle test-unused-modules test-soft_failure-no-reference test-spec test-invalid-syntax
```

Comment:
this is wrong because `test-metadata-changed` is supposed to be called only for uncommited files but `test-static` is supposed to be called on a "commited state" instead like all the other checks.

Reply:
OK, let me try again.